include/ofi_shm.h

```diff
@@ -72,6 +72,7 @@ enum {
 	smr_src_inline,	/* command data */
 	smr_src_inject,	/* inject buffers */
 	smr_src_iov,	/* reference iovec via CMA */
+	smr_src_offset,	/* use inject buffer offsets with resp*/
```

Comment:
Maybe call this SAR instead of offset?  The use of the term offset isn't clear to me.  If I understand the behavior, this is using multiple inject buffers.